catalog-info.yaml

```diff
@@ -630,6 +630,53 @@ spec:
         everyone:
           access_level: BUILD_AND_READ
 
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-beats-xpack-otel
+  description: "Beats x-pack otel pipeline"
+  links:
+    - title: Pipeline
+      url: https://buildkite.com/elastic/beats-xpack-otel
+
+spec:
+  type: buildkite-pipeline
+  owner: group:ingest-fp
+  system: platform-ingest
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: beats-xpack-otel
+      description: "Beats x-pack otel pipeline"
+    spec:
+      branch_configuration: "main 7.17 8.* 9.*"
+      pipeline_file: ".buildkite/x-pack/pipeline.xpack.otel.yml"
+      maximum_timeout_in_minutes: 120
+      provider_settings:
+        trigger_mode: none # don't trigger jobs from github activity
+        build_pull_request_forks: false
+        build_pull_requests: true # requires filter_enabled and filter_condition settings as below when used with buildkite-pr-bot
+        build_tags: true
+        filter_enabled: true
+        filter_condition: >-
+          build.pull_request.id == null || (build.creator.name == 'elasticmachine' && build.pull_request.id != null)
```

Comment:
IIUC, this will not be honoured when using `trigger_mode: none`. Although I see it was included in https://github.com/elastic/beats/pull/37980, the person who did it is no longer part of the team, so I'm certainly confused about those settings.